guides/17_Migration_to_3_0.md

```diff
@@ -162,6 +162,51 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+The following `AudioVideo` events have been removed in v3.x, these events redirect data from `metricDidReceive`.
+
+- videoSendHealthDidChange
+- videoSendBandwidthDidChange
+- videoReceiveBandwidthDidChange
+
+In 3.x you can still replecate these events funtionality through `metricDidReceive`.
+
+
+```js
+const observer = {
+  oldSendBandwidth: 0,
+  oldRecvBandwidth: 0,
+  metricsDidReceive: (clientMetricReport) => {
+    const metricReport = clientMetricReport.getObservableMetrics();
+
+    const videoUpstreamPacketPerSecond = metricReport.videoPacketSentPerSecond;
+    const videoUpstreamBitrate = metricReport.videoUpstreamBitrate;
+    const availableSendBandwidth = metricReport.availableOutgoingBitrate;
+    const availableRecvBandwidth = metricReport.availableIncomingBitrate;
+    const nackCountPerSecond = metricReport.nackCountReceivedPerSecond;
+    // videoSendHealthDidChange
+    console.log(`Sending video bitrate in kilobits per second: ${videoUpstreamBitrate / 1000} and ${videoUpstreamPacketPerSecond}`);
```

Comment:
nit: Do we need to mention that `${videoUpstreamPacketPerSecond}` is video packet per second?

Reply:
Updated